packages/client/src/app/store/index.ts

```diff
@@ -1,15 +1,31 @@
-import { configureStore } from "@reduxjs/toolkit";
+import {
+  configureStore,
+  createSerializableStateInvariantMiddleware,
+} from "@reduxjs/toolkit";
+import { persistReducer, persistStore } from "redux-persist";
+import storage from "redux-persist/lib/storage";
 
 import { gameReducer } from "@/entities/game/model/store/gameSlice";
 import { yandexApi } from "@/shared/api/yandexApi";
 
+const config = {
+  key: "root",
+  storage,
+};
+
+const persistGameReducer = persistReducer(config, gameReducer);
+
 export const store = configureStore({
   reducer: {
     [yandexApi.reducerPath]: yandexApi.reducer,
-    game: gameReducer,
+    game: persistGameReducer,
   },
   middleware: getDefaultMiddleware =>
-    getDefaultMiddleware().concat(yandexApi.middleware),
+    getDefaultMiddleware({ serializableCheck: false }).concat(
```

Comment:
Не надо полностью отключать мидлвару. Надо настройки указать. А конкретно вот так

```suggestion
    getDefaultMiddleware({ serializableCheck: {
        ignoredActions: ['persist/PERSIST']
    } }).concat(
```